### PR TITLE
Enable Ubuntu 16.04 build at packager.io

### PR DESCRIPTION
**Pull request by gianperrone**
This enables building a package for Ubuntu 16.04 LTS (Xenial) as suggested in #1263. I seriously have no idea why bzr and mercurial are needed as build dependencies, but the build only proceeds with them; I got the idea from the packager.io go example project.

The build succeeds: https://packager.io/gh/gianperrone/gogs/build_runs/8#71773
I have a test installation running and found no problems.

### CODE REVIEW

File: .pkgr.yml [hunk 1]
```diff
@@ -7,10 +7,15 @@ targets:
...
   debian-8:
     <<: *debian
+  ubuntu-12.04:
+    <<: *debian
   ubuntu-14.04:
     <<: *debian
-  ubuntu-12.04:
+  ubuntu-16.04:
     <<: *debian
+    build_dependencies:
```
Comment (bkcsoft) [inline .pkgr.yml]: Why were these added? and why not add them to `debian-7` instead as I'm guessing all builds would need them?

Comment (gianperrone) [inline .pkgr.yml]: This is an example of a build without these dependencies: https://packager.io/gh/gianperrone/gogs/build_runs/1#71723

As you can clearly see, not all builds need it. The idea to add these came from looking at the log of the failing build (see link) and from looking at this history: https://github.com/pkgr/example-go-app/commits/master
Apparently bzr and mercurial are needed in all builds, but apart from Ubuntu 16.04 they are installed by default on the build images.

**Comment by MrFloppy** [comment]
Please merge this PR, as I really like to use the package manager for gogs on my ubuntu 16.04.

**Comment by jhasse** [comment]
systemd configuration is still missing though, correct?
